tpot/tpot.py

```diff
@@ -169,17 +169,27 @@ def __init__(self, population_size=100, generations=100,
         self._pset.addPrimitive(self._select_percentile, [pd.DataFrame, int], pd.DataFrame)
         self._pset.addPrimitive(self._rfe, [pd.DataFrame, int, float], pd.DataFrame)
 
-        # Mathematical operators
-        self._pset.addPrimitive(operator.add, [int, int], int)
-        self._pset.addPrimitive(operator.sub, [int, int], int)
-        self._pset.addPrimitive(operator.mul, [int, int], int)
-        self._pset.addPrimitive(self._div, [int, int], float)
+        int_terminals = np.concatenate((np.arange(0, 51, 1),
+                np.arange(60, 110, 10)))
 
-        for val in range(0, 101):
+        for val in int_terminals:
             self._pset.addTerminal(val, int)
-        for val in [100.0, 10.0, 1.0, 0.1, 0.01, 0.001, 0.0001]:
+
+        float_terminals = np.concatenate(([0., 1e-6, 1e-5, 1e-4, 1e-3],
+                np.linspace(0.01, 1.00, 100),
+                np.linspace(2., 50., 49),
+                np.linspace(60., 100., 5)))
+
+        for val in float_terminals:
             self._pset.addTerminal(val, float)
 
+        # Remove this immediately
+        def dummy(foo):
+            return foo
+
+        self._pset.addPrimitive(dummy, [int], int)
+        self._pset.addPrimitive(dummy, [float], float)
```

Comment:
Oops, missed that. That really stinks. I do remember debugging something related to this before...

Do the `dummy` primitives count toward the pipeline size in the fitness evaluation? That could be one way to cut down on them, but at the same time, it would unfairly penalize operators with several parameters. Hrm...